receiver/snmpreceiver/config.go

```diff
@@ -208,7 +208,7 @@ type SumMetric struct {
 type ScalarOID struct {
 	// OID is required and is the scalar OID that is associated with a metric
 	OID string `mapstructure:"oid"`
-	// ResourceAttributes is optional and may contain values of Scalar OIDs to associate this metric with
+	// ResourceAttributes is optional and may contain values of (Scalar OIDs only) to associate this metric with
```

Comment:
minor nit: usually i don't think you use parenthesis in a way where they are an integral part of the sentence (removing them in this case would make the sentence broken). I'd suggest something like "...is optional and may contain only scalar OID values to associate..."